microsoft-365/enterprise/microsoft-365-network-connectivity-principles.md

```diff
@@ -143,52 +141,35 @@ Microsoft is transparent about datacenter security, operational security, and ri
+  - Direct internet access should be prioritized for the Microsoft 365 domains by reducing reliance on wide area network (WAN) backhauling, avoiding network hairpins, and enabling a more efficient internet egress local to the users and directly to the Microsoft network.
+  - Ensure that DNS name resolution occurs close to the network egress to ensure that connections are served through the most optimal Microsoft 365 front door.
+  - Prioritize Microsoft 365 connections along the network path, ensuring capacity and quality of service for Microsoft 365 experiences.
+  - Bypass traffic intermediation devices such as proxies and VPN services.
 
-    Examples of  *Optimize*  endpoints include *https://outlook.office365.com*, *https://\<tenant\>.sharepoint.com*, and *https://\<tenant\>-my.sharepoint.com*.
+Customers with complex network topologies, implementing network optimizations like custom routing, IP based proxy bypass, and split tunnel VPN may require IP prefix information in addition to domains. To facilitate these customer scenarios Microsoft 365 network endpoints are grouped into categories to prioritize and ease the configuration of these additional network optimizations. Network endpoints classified under the **“Optimize”** and **“Allow”** categories carry high traffic volumes and are sensitive to network latency and performance, and customers may want to optimize connectivity to those first. Network endpoints under the **“Optimize”** and **“Allow”** categories have IP addresses listed along with domains. Network endpoints classified under the **“Default”** category do not have IP addresses associated with them as they are more dynamic in nature and IP addresses change over time.
 
-    Optimization methods include:
+### Additional network considerations
+When optimizing connectivity to Microsoft 365, certain network configurations may have a negative impact on Microsoft 365 availability, interoperability, performance, and user experience. Microsoft has not tested the following network scenarios with our services, and they are known to cause connectivity issues.
+  - TLS termination or deep packet inspection of any M365 domains with customer proxies or other types of network devices or services [(Use third-party network devices or solutions with Microsoft 365 - Microsoft 365 | Microsoft Learn)](/troubleshoot/miscellaneous/office-365-third-party-network-devices).
+  - Blocking specific protocols or protocol versions such as QUIC, WebSocket’s, etc. by intermediate network infrastructure or service. 
+  - Forcing downgrade or failover of protocols (such as UDP  TCP, TLS1.3  TLS1.2  TLS1.1) used between client applications and Microsoft 365 services.
```

Comment:
Looks like there are some characters here which wont render on the page. Please confirm. "(such as UDP  TCP, TLS1.3  TLS1.2  TLS1.1)"